packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -29,7 +29,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:2.3.2'
+    return 'defi/defichain:HEAD-edbeedf'
```

Comment:
guess need a latest version
```suggestion
    return 'defi/defichain:HEAD-88459e7'
```

Reply:
Yes you are right! thanks for that